engine/crates/parser-sdl/src/rules/visitor/context.rs

```diff
@@ -219,6 +219,9 @@ impl<'a> VisitorContext<'a> {
             required_udfs.insert((UdfKind::Authorizer, name.clone()));
         }
 
+        // lets make sure we add the global rules to the federated graph config as well
```

Comment:
not entirely happy with this though

Reply:
You could probably avoid it by changing the places that build the config as the parent most likely has the config. But probably not that important for now. 🤷